### DISCUSSION TITLE
Light chart shown in dark mode

### DISCUSSION THREAD

**Discussion by HarshaKosala** [tradingview/lightweight-charts#902] [Offtopic]
Hi,

We are using dark mode of the chart by setting the theme as 'Dark'. But chart showing as below. Toolbars as dark but, chart area as light.
Any help would be highly appreciate.

![image](https://user-images.githubusercontent.com/19972882/142231799-437f45fd-c5bc-4c84-9674-8222519cfc4a.png)

![image](https://user-images.githubusercontent.com/19972882/142231700-6cc3df23-5bb0-49d7-a6c4-9b3f6cdbc193.png)

Thanks

**Comment by timocov** [comment]
@Harsha192 any reason why you opened this discussion in this repo rather than use appropriate repo https://github.com/tradingview/charting_library/ ?
